src/administration/web.md

```diff
@@ -87,3 +105,29 @@ $ platform environment:restore
 ```
 
 to restore an existing snapshot.
+
+### Command Pull-downs
+
+#### URLs
+
+![Header URL](/images/header_url.png)
+
+The URLs pull-down exposes the domains that can be used to access application environments from the web.
+
+#### GIT
+
+![Header Git](/images/header_git.png)
+
+The Git pull-down displays the commands to use to clone the codebase via Git.
+
+#### CLI
+
+![Header CLI](/images/header_cli.png)
+
+The CLI pull-down displays the commands to get your project set up locally with the Platform.sh CLI.
+
+#### SSH
+
+![Header SSH](/images/header_ssh.png)
+
+The SSH pull-down display the commands to access your project with SSH.
```

Comment:
Nit: Over ssh, not with ssh.

Also, I don't know if we're consistent about ssh vs SSH.  Are we?

Reply:
We use SSH everywhere except in four places it looks like. SSH > ssh?

Reply:
Capitalize all the Secure Shells!